pipelines/shopgate.order.createOrder.v1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate.order.createOrder.v1",
+    "public": true,
+    "input": [
+      {"key": "order", "id": "10"}
+    ],
+    "output": [
+      {"key": "orderId", "id": "20"}
+    ],
+    "steps": [
+      {
+        "type": "extension",
+        "id": "@shopgate/order",
+        "path": "@shopgate/order/order/preCheck.js",
```

Comment:
The step name looks a bit cryptic: `preCheck`.

Reply:
- Check if user is loggedin (user storage available)
- ... other pre-conditions logic before process order

Reply:
Changed to `checkContext`